src/measure_theory/set_integral.lean

```diff
@@ -13,42 +13,56 @@ import measure_theory.lebesgue_measure
 
 Integrate a function over a subset of a measure space.
 
-## Main definition
+## Main definitions
 
 `measurable_on`, `integrable_on`, `integral_on`
 
-## Tags
+## Notation
 
-indicator, characteristic
+`∫ a in s, f a` is `measure_theory.integral (s.indicator f)`
 -/
 
 noncomputable theory
-open_locale classical topological_space
-open set lattice filter topological_space ennreal emetric measure_theory
-
-set_option class.instance_max_depth 50
+open set filter topological_space measure_theory measure_theory.simple_func
+open_locale classical topological_space interval
 
 universes u v w
 variables {α : Type u} {β : Type v} {γ : Type w}
 
-section measurable_on
+namespace set
 variables [measurable_space α] [measurable_space β] [has_zero β] {s : set α} {f : α → β}
 
 /-- `measurable_on s f` means `f` is measurable over the set `s`. -/
 @[reducible]
 def measurable_on (s : set α) (f : α → β) : Prop := measurable (indicator s f)
 
-lemma measurable_on_empty : measurable_on ∅ f :=
+@[simp] lemma measurable_on_empty (f : α → β) : measurable_on ∅ f :=
 by { rw [measurable_on, indicator_empty], exact measurable_const }
 
-lemma measurable_on_univ (hf : measurable f) : measurable_on univ f :=
+@[simp] lemma measurable_on_univ (hf : measurable f) : measurable_on univ f :=
```

Comment:
This one should not be in the namespace set, as there is no set, and it should be `measurable.measurable_on_univ`, I think. In fact, most stuff in this section should not be in the namespace set. You should probably remove the `namespace set`, and define `measurable_on` as `set.measurable_on`. Not sure I would put it as reducible, also.